docs/en-US/utilities.md

```diff
@@ -70,6 +70,11 @@ With this in mind, I would expect the java repository mentioned earlier to have
 
 The name of the subfolder maps directly on to what is put in `vvv-custom.yml`. VVV will run the `provision.sh` file, at which point it can do as it pleases. This could be installing a package via `apt-get` or something else. Other files can be included in these folders for `provision.sh` to make use of.
 
+## Utility Provisioning
+
+It's possible to provision a single utility on its own, without provisioning the whole of VVV using the `--provision-with` parameter of vagrant.  
+As an example from the `config.yml` above we can run `vagrant provision --provision-with="utility-core-php56"` to provision only the PHP 5.6 utility.
```

Comment:
One thing. This won't work if we don't push the config.yml file.

Eg: when needing to install a new PHP version.

Reply:
Do you mean that we need to specify that before to execute that command is required to update the config file?

Reply:
if you don't add PHP 5.6 as a utility to install in `config.yml` then this command won't work. There is no provisioner by that name in that scenario.

Reply:
well we are talking about the example f of the config file in that page, so I don't know if we need to add something in the doc.